docs/jwst/extract_1d/arguments.rst

```diff
@@ -45,59 +72,28 @@ The ``extract_1d`` step has the following step-specific arguments.
   0 for that particular column (or row). If "bkg_fit" is not "poly", this
   parameter will be ignored.
 
-``--bkg_sigma_clip``
-  The background values will be sigma-clipped to remove outlier values from
-  the determination of the background. The default value is a 3.0 sigma clip.
-
 ``--log_increment``
-  Most log messages are suppressed while looping over integrations, i.e. when
-  the input is a CubeModel or a 3-D SlitModel.  Messages will be logged while
-  processing the first integration, but since they would be the same for
-  every integration, most messages will only be written once.  However, since
-  there can be hundreds or thousands of integrations, which can take a long
-  time to process, it would be useful to log a message every now and then to
-  let the user know that the step is still running.
-
-  ``log_increment`` is an integer, with default value 50.  If it is greater
-  than 0, an INFO message will be printed every ``log_increment``
-  integrations, e.g. "... 150 integrations done".
+  For multi-integration extractions, if this parameter is set to a value greater
+  than zero, an INFO-level log message will be printed every `log_increment` integrations
+  to report on progress. Default value is 50.
 
-``--subtract_background``
-  This is a boolean flag to specify whether the background should be
-  subtracted.  If None, the value in the :ref:`EXTRACT1D <extract1d_reffile>`
-  reference file (if any) will be used.  If not None, this parameter overrides
-  the value in the reference file.
+``--save_profile``
```

Comment:
Save_profile is only an option for PSF fitting correct ?  Same question about save_scene_model 
So should there be a section for PSF fitting options so that is no confusion

Reply:
No, these can be saved if desired for box or optimal fitting.  Optimal is not yet an option in this version of the code.

For box extractions, the profile shows the pixel weights used while summing over the aperture: 1 for fully included pixels, 0 for pixels not included.  The scene model shows a uniform flux inside the aperture that sums in the cross-dispersion direction to the output spectrum.  For box extractions, the profile can be useful to make sure your aperture is what you intended to set via the input reference file.  The scene model is less helpful in this use case, but might still be diagnostic, if you're trying to figure out why the output spectrum has the values it does.

Reply:
ok got it